safir/src/safir/pydantic/_types.py

```diff
@@ -134,6 +141,7 @@ def _validate_human_timedelta(v: str | float | timedelta) -> float | timedelta:
 SecondsTimedelta: TypeAlias = Annotated[
     timedelta,
     BeforeValidator(lambda v: v if not isinstance(v, str) else int(v)),
+    PlainSerializer(lambda t: t.total_seconds(), return_type=float),
```

Comment:
This confuses me since I was getting `timedelta` objects in the serialization from `model_dump` with the existing setup, even though the Pydantic documentation seems to imply that this wouldn't be the case. But I'll add that anyway to be more explicit.

Reply:
There's also the `ser_json_timedelta` [model config option](https://docs.pydantic.dev/latest/api/config/#pydantic.config.ConfigDict.ser_json_timedelta) which can be set to make all `timedelta` instances serialize to `float` seconds, but that would have to be set on every model (whether explicitly in each, or in a base class model).

Reply:
Ah, okay, I'm not sure what I was doing before, but I am able to reproduce this problem with a proper test. Thanks!